docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -979,19 +979,19 @@ Given any ksqlDB type, returns the equivalent JSON string.
 ```sql title="Primitives types"
 TO_JSON_STRING(1) => "1"
```

Comment:
This example and the next one don't show up in output with double quotes. Should these be single quotes instead?

Reply:
The idea behind these double quotes was to show that the return type is a string. I see how this can be confusing from these examples. Do you think it will be more intuitive to just drop the double quotes? The function description already specifies the return type, so it should be sufficient.